### ISSUE TITLE
Gallery: [Vue warn]: Error in render: "TypeError: Cannot read property 'supportedFeatures' of undefined"

### ISSUE THREAD

**Issue by dbezborodovrp** [FineUploader/vue-fineuploader#36]
I tried using the `Gallery` component as described in the basic example. The gallery did not display and I received the following error in the console:

`[Vue warn]: Error in render: "TypeError: Cannot read property 'supportedFeatures' of undefined"`

The component that uses `<gallery>`:

```
<template>
    <div class="row">
        <div class="col-md-4">
            <gallery :uploader="uploader" />
        </div>
    </div>
</template>

<script>

import FineUploaderTraditional from 'fine-uploader-wrappers';
import Gallery from 'vue-fineuploader/gallery';

export default {
    components: {
        Gallery
    },

    props: ['tower_id'],

    data() {
        const uploader = new FineUploaderTraditional({
            options: {
                request: {
                    endpoint: 'my/upload/endpoint'
                }
            }
        });

        return {
            uploader
        }
    },

    computed: {
    },

    methods: {
    },

    mounted() {
    }
}

</script>
```

Package versions:

- vue 2.5.13
- vue-fineuploader 2.0.0-rc.7
  - fine-uploader 5.15.6
  - fine-uploader-wrappers 0.1.1
  - lodash.isplainobject 4.0.6

**Comment by Elhebert** [comment]
Hi,

🤔  Wierd, I did try it before merging it, and it didn't occur to me, but I'll check this out later today.

Any idea @tjunxiang92?

**Comment by Elhebert** [comment]
I found the error, the `fine-uploader-wrappers` version is outdated. I'll update the dependency version number.

**Comment by Elhebert** [comment]
Fixed in 166758e8a6c2ad02059890797a351e650e8f73b7

Try an `npm update` and it should be fine.

**Comment by dbezborodovrp** [comment]
Works for me.
